src/init.cpp

```diff
@@ -71,6 +75,10 @@ bool fFeeEstimatesInitialized = false;
 static CZMQNotificationInterface* pzmqNotificationInterface = NULL;
 #endif
 
+#if ENABLE_PROTON
+static AMQPNotificationInterface* pAMQPNotificationInterface = NULL;
```

Comment:
Classes beginning with 'C' are from upstream.  We have not been following that consistently e.g. ZCJoinSplit vs JSDescription.  In this case, it became hard to read the code e.g. CAMQPBlahBlah... when I knew I was looking for AMQP related code, e.g. AMQPBlahBlah..., so I thought it would be better to be explicit to try and avoid errors.